internal/caller/servicecaller_test.go

```diff
@@ -40,6 +43,27 @@ func TestMarshalJSON(t *testing.T) {
 	assert.Equal(t, "test", res.Name)
 }
 
+func TestMarshalText(t *testing.T) {
+	req := grpc_testing.SimpleRequest{}
+	dynReq := dynamicpb.NewMessage(req.ProtoReflect().Descriptor())
+
+	responseStatusField := dynReq.Descriptor().Fields().ByName("response_status")
+	rs := &grpc_testing.EchoStatus{}
+	dynRS := dynamicpb.NewMessage(rs.ProtoReflect().Descriptor())
+
+	codeField := dynRS.Descriptor().Fields().ByName("code")
+	dynRS.Set(codeField, protoreflect.ValueOfInt32(1))
+	messageField := dynRS.Descriptor().Fields().ByName("message")
+	dynRS.Set(messageField, protoreflect.ValueOfString("oops"))
+	dynReq.Set(responseStatusField, protoreflect.ValueOf(dynRS))
+
+	sc := NewServiceCaller(nil, JSON, Text, nil, false)
+	res, err := sc.marshalMessage(dynReq)
+	require.NoError(t, err)
+
+	assert.Equal(t, `response_status:{code:1  message:"oops"}`, string(res))
+}
+
 func TestMarshalJSON_AnyNotFound(t *testing.T) {
```

Comment:
I had to change the logic of this test because it now fails directly when we [unmarshal](https://github.com/vadimi/grpc-client-cli/pull/109/files?diff=unified&w=0#diff-79cd544f45d00146ac7737eacc57ef42cc2f91ff39711db2dcc947d0bd287ad6R93) the message (I'm not sure why it succeeded before)